apollo/helpers.ts

```diff
@@ -68,20 +71,38 @@ export const getOrganizationById = async (
   return await models.Organization.findOne({ id });
 };
 
+export const addTalent = async (input: TalentInput): Promise<Talent> => {
+  let newTalent: TalentInput = defaultTalentInput;
+  try {
+    newTalent = await models.Talent.createOne(input);
+  } catch (error) {
+    console.error(error); //eslint-disable-line no-console
+  }
+  return {
+    ...newTalent,
+    fullName: getFullName(newTalent),
+    isBasicInfoComplete: false,
+  };
+};
+
 export const addUser = async (input: UserInput): Promise<User> => {
   const type =
     input.type === 'TALENT'
       ? 'Talent'
       : input.type === 'EMPLOYER'
       ? 'Employer'
       : 'Agency';
+  // unnecessary: check through firebase
   const oldUser = models[type].findOne({
     email: input.email,
   });
   if (oldUser) throw new Error('user already exists');
+  // END: unnecessary: check through firebase;
   return await models[type].createOne(input);
 };
```

Comment:
wasn't the plan to dump addUser now?

Reply:
you're right. I kept it in, since we sort of still need it for the employer bit. I can rename it to addEmployer for now, though.

Reply:
OK. I've removed the addUser mutation and its resolver. I'm keeping the addUser helper for the time being and use it in the addEmployer resolver, since we